index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<header>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+</header>
+
+<body>
+    <nav class="zone sticky crimson">
+        <ul class="navbar">
+            <li><a href="">About</a></li>
+            <li><a href="#projects">Work</a></li>
+            <li><a href="">Contact</a></li>
+        </ul>
+    </nav>
+    <div id="welcome-section">
+        <h1>Welcome to my portfolio</h1>
+    </div>
```

Comment:
- [ ] Instead of using a div tag it's better to use a section  tag to follow HTML semantic guidelines